flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/spargel/VertexUpdateFunction.java

```diff
@@ -40,7 +42,37 @@
 	VertexValue extends Serializable, Message> implements Serializable {
 
 	private static final long serialVersionUID = 1L;
-	
+
+	// --------------------------------------------------------------------------------------------
+	//  Attributes that allow vertices to access their in/out degrees and the total number of vertices
+	//  inside an iteration.
+	// --------------------------------------------------------------------------------------------
+
+	private long numberOfVertices = -1L;
+
+	public long getNumberOfVertices() throws Exception{
+		if (numberOfVertices == -1) {
+			throw new InaccessibleMethodException("The number of vertices option is not set");
+		}
+		return numberOfVertices;
+	}
+
+	void setNumberOfVertices(long numberOfVertices) {
+		this.numberOfVertices = numberOfVertices;
+	}
+
+	//---------------------------------------------------------------------------------------------
+
+	private boolean optDegrees;
+
+	public boolean isOptDegrees() {
```

Comment:
Are you talking about the getters? To my knowledge, these are public. Otherwise, they would be totally unusable. 
If you're talking about the default setters, I needed to see them in the entire package. But they are not public.